init-script

```diff
@@ -218,7 +218,7 @@ usb_setup_configfs() {
     fi
     echo $USB_FUNCTIONS | grep -q "mass_storage" && ln -s $GADGET_DIR/g1/functions/storage.0 $GADGET_DIR/g1/configs/c.1
 
-    echo "$(ls /sys/class/udc)" > $GADGET_DIR/g1/UDC
+    echo "$(ls /sys/class/udc | grep -v dummy | head -1)" > $GADGET_DIR/g1/UDC
```

Comment:
I don't think it's good continue to use ls like this, the change below would be cleaner and also a bit faster:
```suggestion
        for udc in /sys/class/udc/* ; do
        case "$udc" in
            */dummy) true ;;
            *) echo "${udc##*/}" > $GADGET_DIR/g1/UDC ; break ;;
        esac
    done
```
For more why it's problematic read:
https://github.com/koalaman/shellcheck/wiki/SC2010